docker-for-mac/index.md

```diff
@@ -131,6 +133,16 @@ You can set the following runtime options.
   Uncheck this to opt out and prevent auto-send of data. Docker may prompt for
   more information in some cases, even with auto-send enabled.
 
+#### Choose whether to store Docker credentials on Mac keychain (Edge feature)
```

Comment:
Can I keep my heading and just add that? The headings are surfaced in the on-page TOCs so I want to keep them